tv_apps/browser/test/unit/browser_dialog_test.js

```diff
@@ -44,84 +46,34 @@ suite('Browser Dialog >', function() {
 
   suite('Initial state', function() {
     test('should init DOM elements', function() {
-      [
-        'browserDialogBase',
-        'browserDialog',
-        'browserDialogTitle',
-        'browserDialogMsg',
-        'browserDialogInput',
-        'browserDialogInputArea',
-        'browserDialogInputClear',
-        'browserDialogButton',
-        'browserDialogButton1',
-        'browserDialogButton2'].forEach(element => {
+      ['browserDialogBase'].forEach(element => {
         expect(subject[element]).to.be.an('object');
       });
```

Comment:
I think we can directly check browserDialogBase here instead of loop through array for one element.